plugins/nvim-lsp/nvim-lightbulb.nix

```diff
@@ -0,0 +1,93 @@
+{
+  pkgs,
+  lib,
+  config,
+  helpers,
+  ...
+}:
+with lib; {
+  options.plugins.nvim-lightbulb = {
+    enable = mkEnableOption "Enable nvim-lightbulb, showing available code actions";
```

Comment:
just went through a massive change to fix these, mkEnableOption automatically adds "Whether to enable..." to the description so "Enable" shouldn't be part of it. I'll fix it, just leaving it here for future notes